frog/imports/ui/GraphEditor/store/operator.js

```diff
@@ -90,10 +89,22 @@ export default class Operator {
     };
   }
 
-  @computed get dragPoint(): [number, number] {
+  @computed get dragPointTo(): [number, number] {
+    return [this.x + 30, this.y + 30];
+  }
+
+  @computed get dragPointFrom(): [number, number] {
+    return this.dragPointTo;
```

Comment:
Happy for you to try it out as a new pull request, and merge it if it looks good! :)